ui/src/components/Employees/style.module.scss

```diff
@@ -0,0 +1,101 @@
+@import 'src/scss/lib';
+
+.employee-list-container {
+  color: $color-text-dim2;
+  padding: 20px;
+  text-align: left;
+  width: 55%;
+
+  @media(max-width: $md) {
+    margin-top: 100px;
+  }
+
+  @media(max-width: $sm) {
+    margin-top: 60px;
+  }
+}
+
+.employee-content {
+  background: $color-select-active;
+  border-radius: 5px;
+  display: grid;
+  grid-auto-rows: minmax(2em, auto);
+  grid-column-gap: 1em;
+  grid-row-gap: 0;
+  grid-template-columns: 20% 25% 1fr;
+  height: 6em;
+  margin-top: 2em;
+  padding: 15px;
+}
+
+.list-section1 {
+  display: grid;
+  grid-auto-rows: minmax(1em, auto);
+  grid-column: 1;
+  grid-column-gap: 7px;
+  grid-row: 1;
+  grid-row-gap: 8px;
+  grid-template-columns: 40% 1fr;
+  grid-template-rows: 26px 26px;
+}
+
+.employee-headshot-wrapper {
```

Comment:
We are using both `container` and `wrapper` in class names. Both of them mean the same thing (i.e wrapper or container element). I suggest we choose one of these. Consistent names are extremely important.